### DISCUSSION TITLE
Feedback on "For you" feed

### DISCUSSION THREAD

**Discussion by davetron5000** [community/community#21086] [Other Feature Feedback, Questions, & Ideas] [answered]
I clicked "Send Feedback" on the "for you" feed and it brought me here, so hopefully this is the right place/form for feedback.

I don't understand what this feed is trying to accomplish.  It shows me:

* A Lyft repo I've never heard of. I've never interacted with Lyft on GitHub
* A repo from someone I worked with 4 years ago about practicing SQL who I never interact with in GitHub
* two more repos from ex co-workers I have not interacted with on GitHub in public ever and in private for years
* increasingly nonsensical recommendations

I just don't understand what problem this is solving for me.  If there is a way to opt out of it or turn it off or configure it, maybe?  But I just don't discover software by stumbling upon random repos.

**Comment by jfullstackdev** [comment, marked as answer]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
